src/Paprika/Chain/Blockchain.cs

```diff
@@ -429,6 +429,7 @@ private class BlockState : RefCountingDisposable, IWorldState, ICommit, IProvide
         /// </summary>
         private PooledSpanDictionary _preCommit = null!;
 
+        private readonly DelayedMetrics.ICounter<long> _xorMissed;
```

Comment:
Does it have to be interface (so indirect call per call)? Could it be a concrete type?

Reply:
Fixed @benaadams